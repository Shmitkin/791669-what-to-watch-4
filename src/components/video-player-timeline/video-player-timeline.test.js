import React from "react";
import renderer from "react-test-renderer";

import VideoPlayerTimeLine from "./video-player-timeline.jsx";

it(`Should render VideoPlayerTimeLine propperly`, () => {
  const tree = renderer.create(
      <VideoPlayerTimeLine
        progress={50}
        remaining={700}
      />
  );
  expect(tree).toMatchSnapshot();
});
