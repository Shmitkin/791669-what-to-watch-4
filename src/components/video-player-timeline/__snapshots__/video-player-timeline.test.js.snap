// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render VideoPlayerTimeLine propperly 1`] = `
<div
  className="player__controls-row"
>
  <div
    className="player__time"
  >
    <progress
      className="player__progress"
      max="100"
      value={50}
    />
    <div
      className="player__toggler"
      style={
        Object {
          "left": "50%",
        }
      }
    >
      Toggler
    </div>
  </div>
  <div
    className="player__time-value"
  >
    0:11:40
  </div>
</div>
`;
